applications/minotari_console_wallet/src/lib.rs

```diff
@@ -284,7 +284,7 @@ fn get_recovery_seed(
 ) -> Result<Option<CipherSeed>, ExitError> {
     if matches!(boot_mode, WalletBoot::Recovery) && !matches!(wallet_type, Some(WalletType::Ledger(_))) {
         let seed = if let Some(ref seed_words) = cli.seed_words {
-            get_seed_from_seed_words(seed_words)?
+            get_seed_from_seed_words(seed_words, None)?
```

Comment:
I think the passphrase should be an option here as well, and be included in function `prompt_private_key_from_seed_words()`.

Reply:
This is fine I think, its an internal helper function, console wallet does not yes use the encrypted seed words

Reply:
But what if you want to convert a mobile wallet with encrypted seed words to a console wallet?

Reply:
https://github.com/tari-project/tari/issues/6572